openlibrary/templates/account/verify/success.html

```diff
@@ -8,5 +8,5 @@ <h1>$_("Hi, %(user)s", user=account.displayname)!</h1>
 
 <div id="contentBody">
   $ link = "/account/login?redirect=/&username=" + url_quote(account.email)
-  <p>$:_("Yay! Your email address has been verified. Please <a href='%s'>log in to continue</a>.", link)</p>
+  <p>$:_("Yay! Your email address has been verified. Please <a href="%s">log in to continue</a>.", link)</p>
```

Comment:
CAUTION: Outer string vs. inner strings.

Reply:
Fixed this in the latest PR